captum/attr/_core/feature_permutation.py

```diff
@@ -40,8 +50,127 @@ def attribute(  # type: ignore
         additional_forward_args: Any = None,
         feature_mask: Optional[TensorOrTupleOfTensors] = None,
         ablations_per_eval: int = 1,
-        **kwargs: Any
+        **kwargs: Any,
     ) -> TensorOrTupleOfTensors:
+        r"""
+        This attribution method essentially implements the permutation feature
```

Comment:
nit: would you, please, put the main description of the algorithm under the class definition, above line 26 ? We decided to copy main descriptions of the algorithms under the class definition since it looks nicer on the web page.